uui/components/widgets/Tag.module.scss

```diff
@@ -67,75 +67,74 @@
     &.size-18 {
         min-height: 18px;
         min-width: 18px;
-        padding-left: 3px;
-        padding-right: 3px;
+        padding: 2px;
 
         :global(.uui-count_indicator):not(:last-child) {
             margin-right: 3px;
         }
 
         :global(.uui-caption), :global(.uui-input) {
-            padding: 2px 3px;
+            padding: 0 3px;
             line-height: 12px;
             font-size: 12px;
         }
     }
 
     &.size-24 {
-        @include button-horizontal-layout(24px, 1px);
+        min-width: 24px;
         min-height: 24px;
+        padding: 2px;
 
         :global(.uui-caption), :global(.uui-input) {
```

Comment:
why do we have `:global(.uui-input)` in tag related css?

Reply:
fixed